.travis.yml

```diff
@@ -8,19 +8,19 @@ env:
     - mapper_git="https://github.com/StanfordAHA/CGRAMapper.git"
     - cgra_git="https://github.com/StanfordAHA/CGRAGenerator.git"
     - pnr_git="https://github.com/cdonovick/smt-pnr"
-    - smt_git="https://github.com/makaimann/smt-switch" 
+    - smt_git="https://github.com/makaimann/smt-switch"
 
-    - halide_branch="coreir"
-    - coreir_branch="master"
-    - mapper_branch="master"
+    - halide_branch="dev-coreir"
+    - coreir_branch="dev"
+    - mapper_branch="cgralib"
     - cgra_branch="master"
-    - pnr_branch="master"
+    - pnr_branch="newcore"
```

Comment:
I wasn't sure on that. 
If we all merge as is and don't delete the branches, then update to flow to point at master branch.  Only then update the flow to point  back at master branches then we never break the build.